images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -58,7 +58,16 @@ RUN apt-get update \
     # This is the folder containing 'links' to benv and build script generator
     && mkdir -p /opt/oryx
 
-RUN if [ "${DEBIAN_FLAVOR}" = "bullseye" ]; then \
+RUN if [ "${DEBIAN_FLAVOR}" = "bookworm" ]; then \
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
+            libicu72 \
+            libcurl4 \
```

Comment:
It looks like `libcurl8` already exists. Could you please give it a try? Similarly, could we update the other dependencies below to newer versions if they exist? That will likely be our only opportunity to upgrade them (outside of security updated) for .NET 8.